internal/driver/onvifdiscovery_test.go

```diff
@@ -0,0 +1,299 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//
+// Copyright (C) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package driver
+
+import (
+	"testing"
+
+	sdkModel "github.com/edgexfoundry/device-sdk-go/v2/pkg/models"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/models"
+	contract "github.com/edgexfoundry/go-mod-core-contracts/v2/models"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	uuid1 = "1793ddc8-28b0-11ed-a261-0242ac120002"
```

Comment:
you can use some uuid generator instead of this?

Reply:
I would but Anthony suggested that I make these constants, as they are reused in multiple tests. I don't think I can use uuid.new() to create a constant, it was giving me errors.